cincinnati/src/plugins/macros.rs

```diff
@@ -36,6 +36,18 @@ macro_rules! get_multiple_values {
     }
 }
 
+#[macro_export]
+macro_rules! new_plugin {
+    ($x:expr) => {
+        Box::new($x)
```

Comment:
I think at this point we don't even need the macro anymore (but I'm fine either ways).

Reply:
That's true, it doesn't help if the type is stable. It was tremendously helpful during development. If there's only a slight chance that we'll change the type in the future, I'd prefer to keep it.

Reply:
Ack, fine for me.